homework/8.js

```diff
@@ -0,0 +1,59 @@
+/* 8.1 First declare at least four variables and assign them different data types.
+8.2 For each variable write a console.log statement that logs the value */
+
+let myFavoriteFruit = "Banana";//string
+console.log (myFavoriteFruit);
+let myFavoriteNumber = 2;//number
+console.log (myFavoriteNumber);
+let myFavoriteCars = ["Mercedes", "Aston Martin"];//array
+console.log (myFavoriteCars);
+let myFavoriteBoolean = true; //boolean
+console.log (myFavoriteBoolean);
+
+/* 8.3 Now write a console.log statement wherein you first explain 
+in words what you think the type of your variables is.*/
+
+console.log ("The type of myFavoriteFruit is: String");
+console.log ("The type of myFavoriteNumber is: number");
+console.log ("The type of myFavoriteCars is: array");
+console.log ("The type of myFavoriteBoolean is: boolean");
+
```

Comment:
Correct answer. Why do you use String and number, array, boolean and not String,  Number, Array, Boolean? string is primitive and String is the Object. Maybe useful to take a look into the difference between those 2.